extensions/amp-lightbox-gallery/0.1/lightbox-caption.js

```diff
@@ -39,14 +39,18 @@ export class LightboxCaption {
     const el = htmlFor(doc)`
       <div class="i-amphtml-lbg-caption">
         <div class="i-amphtml-lbg-caption-scroll">
-          <div class="i-amphtml-lbg-caption-text"></div>
+          <div class="i-amphtml-lbg-caption-text amp-lightbox-gallery-caption"></div>
         </div>
         <div class="i-amphtml-lbg-caption-mask"></div>
       </div>`;
     return new LightboxCaption(
       el,
       dev().assertElement(el.querySelector('.i-amphtml-lbg-caption-scroll')),
-      dev().assertElement(el.querySelector('.i-amphtml-lbg-caption-text')),
+      dev().assertElement(
+        el.querySelector(
+          '.i-amphtml-lbg-caption-text.amp-lightbox-gallery-caption'
```

Comment:
This can target the `.amp-lightbox-gallery-caption` class exclusively for the same effect.